wp_bak/app/wp_bak.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+# Docker:
+# docker build -t colindavey/wp_bak:alpha .
+# docker run -v `pwd`/src:/home/mvp/app/src -v `pwd`/dst:/home/mvp/app/dst -t colindavey/wp_bak:alpha
+
+from datetime import datetime
+from fnmatch import fnmatch
+import logging
+import os
+import shutil
+import sys
+import tarfile
+import time
+
+#############################
+# CONSTANTS
+#############################
+
+SRC_DIR='src/'
+DST_DIR='dst/'
+SHORT_DIR=DST_DIR+'shorts/'
+LONG_DIR=DST_DIR+'longs/'
+
+MINUTE = 60
+HOUR = 60 * MINUTE
+DAY = 24 * HOUR
+WEEK = 7 * DAY
+
+DATE_TIME_FORMAT = "%Y-%m-%d-%H-%M-%S"
+
+# Duration between backups in seconds
+# Actual value
+# SHORT_DURATION = DAY
+# Value for testing
```

Comment:
Consider using command line options or environment variables and defaults. Then to test you'd pass the command line argument to override the default (e.g. `wp_bak.py --short_duration=2` for testing)

Reply:
Would you suggest the argument be in seconds?

Reply:
I think the normal case is hours or days so I'd do that. You also want something much quicker for testing so maybe just `--test_mode` flag that sets the durations to a few seconds?